addons/hr/models/hr_employee.py

```diff
@@ -283,6 +277,16 @@ def _compute_presence_icon(self):
             employee.hr_icon_display = 'presence_' + employee.hr_presence_state
             employee.show_hr_icon_display = bool(employee.user_id)
 
+    @api.model
+    def _get_certificate_selection(self):
+        return [
+            ('graduate', _('Graduate')),
+            ('bachelor', _('Bachelor')),
+            ('master', _('Master')),
+            ('doctor', _('Doctor')),
+            ('other', _('Other')),
```

Comment:
We are gradually moving away from `_()` in favor of `self.env._()`, so I would recommend using that.
```suggestion
            ('graduate', self.env._('Graduate')),
            ('bachelor', self.env._('Bachelor')),
            ('master', self.env._('Master')),
            ('doctor', self.env._('Doctor')),
            ('other', self.env._('Other')),
```